src/main/java/com/adyen/model/checkout/InputDetail.java

```diff
@@ -64,12 +64,7 @@ public InputDetail configuration(Map<String, String> configuration) {
     }
 
     public InputDetail putConfigurationItem(String key, String configurationItem) {
-
-        if (this.configuration == null) {
-            this.configuration = null;
```

Comment:
I think this line was wrong before and should be fixed, not removed.
If configuration is null, a new map should be created so it prevents nullPointerExceptions to happen on the next lines
`configuration = new HashMap<>();`

Reply:
Fixed.